src/Bicep.Core/CoreResources.resx

```diff
@@ -343,12 +343,26 @@
   <data name="SecureParamsInNestedDeployRule_Solution" xml:space="preserve">
     <value>Either set the deployment's properties.expressionEvaluationOptions.scope to 'inner' or use a Bicep module instead.</value>
   </data>
-  <data name="UseParentPropertyRuleDescription" xml:space="preserve">
+  <data name="UseResourceSymbolReferenceRule_Description" xml:space="preserve">
+    <value>Use a direct resource symbol reference instead of 'reference' or 'list*' functions.</value>
+  </data>
+  <data name="UseResourceSymbolReferenceRule_CodeFix" xml:space="preserve">
+    <value>Use a direct resource reference.</value>

```

Comment:
Fix titles are commands and commands should not end with a period.
```suggestion
    <value>Use a direct resource reference</value>

```

Reply:
See also https://github.com/Azure/bicep/pull/9909

Reply:
Good point, and thanks for adding validation to catch this in future!